pkg/instrumentation/podmutator.go

```diff
@@ -31,6 +32,11 @@ import (
 var (
 	errMultipleInstancesPossible = errors.New("multiple OpenTelemetry Instrumentation instances available, cannot determine which one to select")
 	errNoInstancesAvailable      = errors.New("no OpenTelemetry Instrumentation instances available")
+
+	EnableDotnetAutoInstrumentationSupport = featuregate.GlobalRegistry().MustRegister(
+		"operator.autoinstrumentation.dotnet",
+		featuregate.StageBeta,
+		featuregate.WithRegisterDescription("controls whether the operator supports .NET auto-instrumentation"))
```

Comment:
Where is this description used/rendered?

Reply:
I wonder if it would be possible to keep the registration of the featuregates in `featuregate` package? 

https://github.com/open-telemetry/opentelemetry-operator/blob/main/pkg/featuregate/featuregate.go#L30

Reply:
Good idea, moved.  For the description, it is only added to the gate, not provided to the FlagSet, so I guess technically we don't need it unless we plan to use it somewhere.

Reply:
Added the feature-gates to our Info log so that users can see the values of each feature gate